### ISSUE TITLE
tflite_convert: command not found

### ISSUE THREAD

**Issue by euDominic** [tensorflow/tensorflow#59260]
I have Ubuntu 20.04 running on a WSL. with conda
NVIDIA Tenserflow 1.15 is installed on it.

As git model I have the TensorFlow Models v.1.13.0 release
https://github.com/tensorflow/models/tree/r1.13.0

now i created a custom model and wanted to convert to tflite.
 but i get the error.....

**------------------------>tflite_convert: command not found<-----------------------------**

what could be the reason? or how can i fix it?

it has to stay at version 1.15, because i have to create models for a Coral EdgeTPU.

pip list

Package Version
------------------------ --------------
absl-py 1.4.0
argon2-cffi 21.3.0
argon2-cffi-bindings 21.2.0
astor 0.8.1
astunparse 1.6.3
async-generator 1.10
attrs 22.2.0
backcall 0.2.0
bleach 4.1.0
cachetools 4.2.4
certifi 2022.12.7
cffi 1.15.1
charset-normalizer 2.0.12
cloudpickle 2.2.0
cycler 0.11.0
cython 0.29.33
dataclasses 0.8
decorator 5.1.1
defusedxml 0.7.1
entrypoints 0.4
guest 0.3.3
google-auth 2.16.0
google-auth-oauthlib 0.4.6
google-pasta 0.2.0
grpcio 1.48.2
h5py 2.10.0
idna 3.4
importlib-metadata 4.8.3
ipykernel 5.5.6
ipython 7.16.3
ipython-genutils 0.2.0
ipywidgets 7.7.2
jedi 0.17.2
jinja2 3.0.3
jsonschema 3.2.0
jupyter 1.0.0
jupyter-client 7.1.2
jupyter-console 6.4.3
jupyter-core 4.9.2
jupyterlab-pygments 0.1.2
jupyterlab-widgets 1.1.1
keras-applications 1.0.8
keras-preprocessing 1.1.2
kiwisolver 1.3.1
lxml 4.9.2
Markdown 3.3.7
MarkupSafe 2.0.1
matplotlib 3.3.4
mistune 0.8.4
nbclient 0.5.9
nbconvert 6.0.7
nbformat 5.1.3
nest-asyncio 1.5.6
notebook 6.4.10
numpy 1.19.5
nvidia-cublas-cu11 11.11.3.6
nvidia-cuda-cupti-cu11 11.8.87
nvidia-cuda-nvcc-cu11 11.8.89
nvidia-cuda-runtime-cu11 11.8.89
nvidia-cudnn-cu11 8.7.0.84
nvidia-cufft-cu11 10.9.0.58
nvidia-curand-cu11 10.3.0.86
nvidia-cusolver-cu11 11.4.1.48
nvidia-cusparse-cu11 11.7.5.86
nvidia-dali-cuda110 1.20.0
nvidia-dali-nvtf-plugin 1.20.0+nv22.12
nvidia-horovod 0.26.1+nv22.12
nvidia-nccl-cu11 2.16.2
nvidia-pyindex 1.0.9
nvidia-tensorflow 1.15.5+nv22.12
oauthlib 3.2.2
object-detection 0.1
opt-einsum 3.3.0
packaging 21.3
pandas 1.1.5
pandocfilters            1.5.0
parso                    0.7.1
pathlib                  1.0.1
pexpect                  4.8.0
pickleshare              0.7.5
Pillow                   8.4.0
pip                      21.2.2
prometheus-client        0.15.0
prompt-toolkit           3.0.36
protobuf                 3.19.6
psutil                   5.9.4
ptyprocess               0.7.0
pyasn1                   0.4.8
pyasn1-modules           0.2.8
pycparser                2.21
Pygments                 2.14.0
pyparsing                3.0.9
pyrsistent               0.18.0
python-dateutil          2.8.2
pytz                     2022.7
PyYAML                   6.0
pyzmq                    25.0.0
qtconsole                5.2.2
QtPy                     2.0.1
requests                 2.27.1
requests-oauthlib        1.3.1
rsa                      4.9
scipy                    1.5.4
Send2Trash               1.8.0
setuptools               59.6.0
six                      1.16.0
sklearn                  0.0.post1
slim                     0.1
tensorboard              2.10.1
tensorboard-data-server  0.6.1
tensorboard-plugin-wit   1.8.1
tensorflow-estimator     1.15.1
tensorrt                 8.5.2.2
termcolor                1.1.0
terminado                0.12.1
testpath                 0.6.0
torch                    1.10.2
tornado                  6.1
traitlets                4.3.3
typing_extensions        4.1.1
urllib3                  1.26.14
utils                    1.0.1
wcwidth                  0.2.5
webencodings             0.5.1
Werkzeug                 2.0.3
wheel                    0.37.1
widgetsnbextension       3.6.1
wrapt                    1.14.1
zipp                     3.6.0
